src/hooks/useTruncate.js

```diff
@@ -45,7 +45,7 @@ const useTruncate = (elementID, truncate, linesBeforeTruncate) => {
         if (linesBeforeTruncate === 1) {
           simpleClamp(element);
         } else {
-          lineClamp(element, { clamp: linesBeforeTruncate });
+          lineClamp.clamp(element, { clamp: linesBeforeTruncate });
```

Comment:
@fernandoabolafio 
This is related to another PR we merged lately https://github.com/decred/pi-ui/pull/194 - which causing an error in pigui !
This change fixes the issue in pigui but re introduced the same error we tried to fix in https://github.com/decred/pi-ui/pull/194, any ideas what may cause this inconsistent behavior ? :thinking:

Reply:
I ended up adding a check to handle both cases.
 - In case the imported var is function I call it directly (works on pi-ui).
 - In case it's an object exposing clamp function then I call the clamp property (works on pigui).

Not sure why the import statement behaves differently in each environment.